common/static/common/js/spec/components/feedback_spec.js

```diff
@@ -1,387 +1,383 @@
-// Generated by CoffeeScript 1.6.1
```

Comment:
I don't feel too strongly about this file, as I've never encoutered it before, and don't expect we'll be back in here any time soon.